lib/sycamore/sycamore/transforms/aryn_partitioner.py

```diff
@@ -0,0 +1,73 @@
+from typing import BinaryIO
+from collections.abc import Mapping
+import requests
+import json
+from sycamore.data.element import create_element, Element
+from typing import List
+import base64
+from tenacity import retry, retry_if_exception_type, wait_exponential, stop_after_delay
+
+_DEFAULT_ARYN_PARTITIONER_ADDRESS = "https://api.aryn.cloud/v1/document/partition"
+_ARYN_PARTITIONING_SERVICE_WAIT_MESSAGE = '{"detail":"Please try again in a little while."}'
+_TEN_MINUTES = 600
+
+
+class ArynPDFPartitionerException(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+
+
+class ArynContinuePDFPartitionerException(Exception):
+    def __init__(self, message):
+        super().__init__(message)
```

Comment:
Instead of having two classes, you can just have a flag in  ArynPDFPartitionerException : can_retry: bool